test/ngResource/resourceSpec.js

```diff
@@ -1544,7 +1544,13 @@ describe('cancelling requests', function() {
       }
     });
 
-    CreditCard.get();
+    var creditCard = CreditCard.get();
+
+    creditCard.$promise.catch(function(err) {
```

Comment:
It would be better to also verify that the function has been called (e.g. by using a spy).
(Here and in the tests below.)